pkg/config/env/netrc.go

```diff
@@ -0,0 +1,15 @@
+package env
+
+import (
+	"os"
+)
+
+// NETRCPath tells you where the .netrc path initially resides.
+// This is so that you can mount the .netrc file to a secret location
+// in the fs system and then move it ~/.netrc. In certain deployments
+// like Kubernetes, we can't mount directly to ~ because it would then
+// clean out whatever is already there as part of the image (such as
+// .cache directory in the Go image).
+func NETRCPath() string {
+	return os.Getenv("ATHENS_NETRC_PATH")
```

Comment:
perhaps this should have used `envy`?

Reply:
@arschles a little intentional since I don't want to default to any value if the env var is not set. I can potentially do `envy.Get("ATHENS_NETRC_PATH", "")` if you'd like? no biggie on my side